worm:
 joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 1000
 joint_wheel_1_1_1_controller:
  type: effort_controllers/JointEffortController
  joint: joint_wheel_1_1_1
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_wheel_1_1_2_controller:
  type: effort_controllers/JointEffortController
  joint: joint_wheel_1_1_2
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_wheel_1_2_1_controller:
  type: effort_controllers/JointEffortController
  joint: joint_wheel_1_2_1
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_wheel_1_2_2_controller:
  type: effort_controllers/JointEffortController
  joint: joint_wheel_1_2_2
  pid: {p: 0.0, i: 0.00, d: 0.0}
 pivot_joint_1_1_controller:
  type: effort_controllers/JointEffortController
  joint: pivot_joint_1_1
  pid: {p: 1.0, i: 0.00, d: 0.0}

